test/setchplenv/verify_setchplenv_scripts.py

```diff
@@ -134,21 +134,24 @@ def get_path_var(var_name):
         path_parts = get_path_var('PATH')
         self.assertTrue(len(path_parts) >= 2)
 
-        path_chpl = path_parts[0].replace(self.chpl_home, 'CHPL_HOME').replace(
-            self.chpl_platform, 'CHPL_PLATFORM')
-        path_util = path_parts[1].replace(self.chpl_home, 'CHPL_HOME')
+        expected_path_chpl = os.path.join(self.chpl_home, 'bin', self.chpl_platform)
+        expected_path_util = os.path.join(self.chpl_home, 'util')
 
-        self.assertEqual('CHPL_HOME/bin/CHPL_PLATFORM', path_chpl)
-        self.assertEqual('CHPL_HOME/util', path_util)
+        actual_path_chpl = path_parts[0]
+        actual_path_util = path_parts[1]
+
+        self.assertEqual(os.stat(expected_path_chpl), os.stat(actual_path_chpl))
+        self.assertEqual(os.stat(expected_path_util), os.stat(actual_path_util))
```

Comment:
os.stat() follows symlinks, so as long as the both sides point at the same directory (with or without symlinks), they should have the same inode values. I ran the verification step on cygwin, though, just to make sure and it worked.

From the python docs, "This function follows symlinks; to stat a symlink use lstat()."

https://docs.python.org/2/library/os.html#os.stat